src/pages/cats.js

```diff
@@ -260,67 +268,84 @@ const Cats = ({ pageContext }) => {
         <Helmet>
           <link rel="canonical" href={canonicalURL} />
         </Helmet>
-        <Section wrap={'reverse'} fullWidth className={cx.SectionHeight}>
-          <AliceCarousel
-            autoPlay
-            animationType="fadeout"
-            autoPlayStrategy="none"
-            items={mockImagesArray}
-            animationDuration={1000}
-            autoPlayInterval={2500}
-            disableButtonsControls
-            disableDotsControls
-            disableSlideInfo
-            infinite
-          />
-          <div
-            style={{
-              flex: 1,
-              display: 'flex',
-              flexDirection: 'column',
-              justifyContent: 'flex-start',
-              minWidth: 220,
-              marginTop: 20,
-              marginBottom: 40,
-              marginLeft: 0,
-              marginRight: 40,
-              alignItems: 'center',
-            }}
-          >
-            <h1 className={cx.title}>Tab For Cats</h1>
-            <p style={{ width: '80%', textAlign: 'center', fontSize: '25px' }}>
-              Open browser tabs, help shelter cats get adopted!
-            </p>
-            <div style={{ marginTop: 20, marginBottom: 20 }}>
-              {installButton}
-            </div>
-            <div style={{ margin: 0 }}>
-              <div>
-                <Star style={{ color: '#ffc533', width: 18, height: 18 }} />
-                <Star style={{ color: '#ffc533', width: 18, height: 18 }} />
-                <Star style={{ color: '#ffc533', width: 18, height: 18 }} />
-                <Star style={{ color: '#ffc533', width: 18, height: 18 }} />
-                <StarHalf style={{ color: '#ffc533', width: 18, height: 18 }} />
-              </div>
+        <div className={cx.SectionHeight}>
+          {isReferral ? (
```

Comment:
Design is awkward. Make it full width and either pin to the top (so it doesn't reflow content on mount) or below the main content on the first page?

![Screen Shot 2021-04-13 at 4 47 29 PM](https://user-images.githubusercontent.com/5924325/114634696-216e4d80-9c91-11eb-8b6b-06e13b5a2748.png)

Reply:
kept some side margin but pinned to top.  Felt like the best look